src/components/Popover.vue

```diff
@@ -29,7 +29,19 @@ interface ElementPosition {
 @description
 Implement a popover component
 
-@params {Boolean} [active = false] toggles visibility
+Use the popover inside a "safe" parent.
```

Comment:
What do you mean by safe?

Reply:
A parent that should never be removed... I should specify that instead of the not clear word "safe" :)